### PR TITLE
Updated the user-interface-samples README. Created the canonical layo…

### PR DESCRIPTION
**Pull request by JonEckenrode**
Updated the user-interface-samples README. Created the canonical layouts README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
-Android User Interface Samples Repository
-=========================================
+# Android user interface samples
 
-This repository contains a set of individual Android Studio projects to help you get
-started writing/understanding Android user interface features.
+A collection of Android Studio projects to help you start building Android user interface features.
```
Comment (alexvanyo) [inline README.md:3]: Probably can remove "Studio"? Calling them an Android Studio project sounds a bit strange, even if that's the tool to edit them with.

```suggestion
A collection of Android projects to help you start building Android user interface features.
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+## Installation
+
+Clone the repository, then open the samples as projects in Android Studio.
+
+## Support
+
+Stack Overflow: http://stackoverflow.com/questions/tagged/android
+
+If you find an error in the samples, please file an issue at https://github.com/android/user-interface.
```
Comment (alexvanyo) [inline README.md:13]: Should we update the link here to be the current `https://github.com/android/user-interface-samples`? It looks this one works, but redirects.

File: CanonicalLayouts/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Installation
+
+Clone the [user-interface-samples](https://github.com/android/user-interface-samples) repository, then open the canonical layout samples as projects in Android Studio.
+
+## Support
+
+Stack Overflow: http://stackoverflow.com/questions/tagged/android
+
+If you find an error in the samples, please file an issue at https://github.com/android/user-interface.
```
Comment (alexvanyo) [inline CanonicalLayouts/README.md:15]: Same here for the link:
```suggestion
If you find an error in the samples, please file an issue at https://github.com/android/user-interface-samples.
```

**Review by alexvanyo — APPROVED** [review]
Just two small items!

File: README.md (hunk 2 above)
Comment (JonEckenrode) [reply README.md]: Revised.

**Pull request merged**